.github/workflows/performance.yml

```diff
@@ -0,0 +1,27 @@
+name: "Performance"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+env:
+  fail-fast: true
+
+jobs:
+  performance-tests:
+    name: "Tests for the performance testing the PDF parsing"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php:
+          - "7.4"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Test dictionary cache"
```

Comment:
Please change name, because related file may run multiple tests in the future. `Run performance tests` or something like that might be a good replacement.